frontend/test/metabase/scenarios/visualizations/pivot_tables.cy.spec.js

```diff
@@ -272,7 +273,9 @@ describe("scenarios > visualizations > pivot tables", () => {
             /Embed this (question|dashboard) in an application/,
           ).click();
           cy.findByText(test.subject);
-          assertOnPivotFields();
+          cy.get("iframe")
+            .its("0.contentDocument.body")
+            .within(assertOnPivotFields);
```

Comment:
@nemanjaglumac Is there another way around this? I got the "0.contentDocument.body" part from [this blog post](https://www.cypress.io/blog/2020/02/12/working-with-iframes-in-cypress/).

Reply:
Before I read the blog post - Why was this change needed in the first place? Was it failing with `iframe[0].src`?

Reply:
I didn't read the blog post I just skimmed for that snippet haha. 

The failure was on the first assertion in `assertOnPivotFields`: `cy.findByText(/Users? → Source/)`. That was weird since it was clearly visible in the page.

Reply:
Oh I missed the `iframe[0].src` part in next test. That approach would probably work in the embed preview one too. I'll push that change if it works.

Reply:
I dug in a little more. I think we want to use both approaches, but I added comments to explain.

For published embeds we need to visit the src to ensure we don't use the preview endpoints. For previews we need to reach inside while it's still an iframe so that it does use the preview endpoints.